benchmarks/Benchmarks.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp3.1;net462</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;netcoreapp2.1;net462</TargetFrameworks>
```

Comment:
```suggestion
    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net462</TargetFrameworks>
```

Reply:
@reyang Just curious, what's going on with this request? Good to have them sorted low-to-high?

Reply:
Yep, as we add more version, we probably wish to append instead of pre-pend the target.
I think we might want to extract these versions to a common `*.props` file before GA (adding `net5` to all projects could be a tedious job).